clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5787,6 +5787,11 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     CmdArgs.push_back("-treat-scalable-fixed-error-as-warning");
   }
 
+  // Enable local accessor to shared memory pass for SYCL.
+  if (isa<BackendJobAction>(JA) && IsSYCL) {
+    CmdArgs.push_back("-mllvm");
+    CmdArgs.push_back("-sycl-enable-local-accessor");
+  }
```

Comment:
A test should be added to verify this is being passed under conditions required.

Reply:
Sure, see here: https://github.com/intel/llvm/pull/5149/commits/a49c7514e3f7e3c2449def223397e50ddfca595d